README.md

```diff
@@ -90,7 +90,7 @@ are welcome.
 
 ## Compatibility
 
-Tested on 1.9.3, 2.0.x and 2.1.x with Exiv2 0.24
+Tested on 1.9.3, 2.0.x and 2.1.x with Exiv2 0.27.1
```

Comment:
The build is failing because 1.9.3 is not supported in travis. Suggest we drop all these and just support 2.4, 2.5 and 2.6

Reply:
Done in 369f319 :) Thanks